soc/silabs/silabs_siwx91x/siwg917/nwp.c

```diff
@@ -36,9 +36,12 @@ int siwx91x_get_nwp_config(sl_wifi_device_configuration_t *get_config, uint8_t w
 		.region_code = DEFAULT_REGION,
 		.boot_option = LOAD_NWP_FW,
 		.boot_config = {
-			.feature_bit_map = SL_SI91X_FEAT_SECURITY_OPEN | SL_SI91X_FEAT_WPS_DISABLE,
+			.feature_bit_map = SL_SI91X_FEAT_SECURITY_PSK | SL_SI91X_FEAT_WPS_DISABLE |
```

Comment:
Are we still able to connect to an open network without `SL_SI91X_FEAT_SECURITY_OPEN`?